src/components/Table/Table.tsx

```diff
@@ -127,6 +129,7 @@ export const Table = <T extends Record<string, any>>({
 	onManualSortBy,
 	showPagination = true,
 	draggableRows = false,
+	resizableColumns = true,
```

Comment:
per our conversation in standup, this should default to false
```suggestion
	resizableColumns = false,
```

Reply:
Updated.